templates/ConfigurationDocumentation.mustache

```diff
@@ -254,6 +254,35 @@
 }</pre>
     </section>
 
+	<section>
+		<h3 id="IndexAllProperties">Index All Properties</h3>
+
+		<p>
+			By default the extension will index only values for properties for which facets are configured.
+			Via this setting, you can make the extension index values for all properties instead.
+		</p>
+
+		<p>
+			Indexing values for all properties is useful
+			when you want to be able to run structured queries for properties not shown in the UI. It is also
+			useful to avoid having to rebuild the search index whenever you add a new property to the UI.
```

Comment:
> add a new property to the UI.

I think this can be misunderstood. I get this refers to the faceted search UI config, but I can see how this might look like it's referring to the general UI (including Wikibase).

Adding an existing property (already indexed) in the faceted search config works automatically. But when a new property is added on the wiki and the config, then the index must still be rebuilt.

The hook for defining the fields to be indexed is separate from the hook returning the data.